### PR TITLE
Only display extra remotes message if needed

### LINKED ISSUE #53: Only mention adding a repo when that's true
The “Installing this app may add a repository of other apps that will show up in AppCenter.” text should only show up when that's actually true. It looks like GNOME Software has logic to determine this, and shows a warning when that's the case.

### PR DESCRIPTION
**Pull request by davidmhewitt**
Fixes #53 

Wasn't sure that concatenating two strings would be a good idea for i18n, so there's a bit of string duplication where we swap one out for the other.

### CODE REVIEW

**Comment by cassidyjames** [comment]
I _think_ concatenating them should be fine, and I could see us wanting to switch up how this is displayed in the future. In fact, I wonder if changing the implications over to a bulleted list could make sense?


i.e. something like:

>### Install untrusted app?
>
>This app is provided solely by its developer and has not been reviewed for security, privacy, or system integration. 
>
> - Download size may be up to 1.2 GB ⓘ
> - Updates to this app will not be reviewed
> - Installing will add a repository that may contain other apps

Of course the concern would be if this dialog gets too unweildy, but I think it might be easier to digest than a big paragraph. Thoughts, @danrabbit?

**Comment by cassidyjames** [comment]
@davidmhewitt oh also: do you have an example of a `.flatpakref` that _doesn't_ contain a remote so I can test this?

**Comment by danrabbit** [comment]
Yeah this is a much smarter design imo and more clearly communicates the implications of the install. Nice work!

**Comment by davidmhewitt** [comment]
I don't know of any ref files that don't include a remote for runtime deps. I've just been testing by removing the flathub remote and checking the message shows and that it doesn't show when I already have the flatpak remote.

**Comment by cassidyjames** [comment]
@davidmhewitt ooooh that makes sense. No sense in warning about a remote that's already been added.

File: src/Views/MainView.vala [hunk 1]
```diff
@@ -77,10 +96,43 @@ public class Sideload.MainView : AbstractView {
...
         if (size != null) {
-            download_size_label.label = _("Download size up to: %s").printf (size);
-            details_stack.visible_child_name = "details";
+            download_size_label.label = _("Download size may be up to %s").printf (size);
+        } else {
+            download_size_label.label = _("Unknown download size");
         }
+
+        if (extra_repo) {
+            var updates_icon = new Gtk.Image.from_icon_name ("system-software-update-symbolic", Gtk.IconSize.BUTTON);
```
Comment (cassidyjames) [inline src/Views/MainView.vala]: I think this should actually always be displayed. Even if the repo is already added, any updates to this app from existing repos will still not be reviewed.

**Comment by davidmhewitt** [comment]
I guess one thing to keep in mind is how we're going to display the sandbox permissions an app will need. I guess we could continue with this bullet pattern, but it may get lengthy?

**Comment by cassidyjames** [comment]
@davidmhewitt yeah, we can figure out if we can summarize it succinctly for bullet(s), or if we'd want another view, or what. But I think this is a big improvement and conveys really important information up front.

**Pull request merged**